### PR TITLE
Configinstate

### PR DESCRIPTION
**Pull request by nicholaspellizer**
This changes provide saving feature for the hyper mode and light\dark theme.

### CODE REVIEW

File: script/engine.js [hunk 1]
```diff
@@ -476,16 +478,29 @@
 			if (darkCss == null) {
 				$('head').append('<link rel="stylesheet" href="css/dark.css" type="text/css" title="darkenLights" />');
 				$('.lightsOff').text(_('lights on.'));
+                $SM.set('config.lightsOff', true, true);
```
Comment (Continuities) [inline script/engine.js]: Nitpick: Indenting is off a bit.

File: script/engine.js [hunk 2]
```diff
@@ -476,16 +478,29 @@
...
 			} else if (darkCss.disabled) {
 				darkCss.disabled = false;
 				$('.lightsOff').text(_('lights on.'));
+    			$SM.set('config.lightsOff', true,true);
```
Comment (Continuities) [inline script/engine.js]: Nitpick: Indenting is off a bit.

File: script/engine.js [hunk 3]
```diff
@@ -476,16 +478,29 @@
...
 			} else {
 				$("#darkenLights").attr("disabled", "disabled");
 				darkCss.disabled = true;
 				$('.lightsOff').text(_('lights off.'));
+                $SM.set('config.lightsOff', false, true);
```
Comment (Continuities) [inline script/engine.js]: Nitpick: Indenting is off a bit.

**Comment by skj3gg** [comment]
Merged in 8320156f42679b47734a80fd51ce65c0f08d8a3e.

I amended commit to fix the indentation (spaces vs tabs)

THANKS!

**Pull request closed without merging**